helm/node-exporter-app/templates/servicemonitor.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ template "prometheus-node-exporter.fullname" . }}
-  namespace: {{ template "prometheus-node-exporter.monitor-namespace" . }}
+  namespace: {{ .Values.prometheus.monitor.namespace }}
```

Comment:
This change is not necessary as `prometheus-node-exporter.monitor-namespace` is set with  `prometheus.monitor.namespace` in the helpers

Reply:
In the helpers there is the following : 
```yaml
{{- define "prometheus-node-exporter.namespace" -}}
  {{- if .Values.namespaceOverride -}}
    {{- .Values.namespaceOverride -}}
  {{- else -}}
    {{- .Release.Namespace -}}
  {{- end -}}
{{- end -}}
```

This leads to having the sm in the `kube-system` ns that's why this change is necessary